packages/vm/docs/README.md

```diff
@@ -6,14 +6,56 @@
 
 ### Modules
 
-* ["evm/evm"](modules/_evm_evm_.md)
-* ["exceptions"](modules/_exceptions_.md)
-* ["index"](modules/_index_.md)
-* ["runBlock"](modules/_runblock_.md)
-* ["runBlockchain"](modules/_runblockchain_.md)
-* ["runCall"](modules/_runcall_.md)
-* ["runCode"](modules/_runcode_.md)
-* ["runTx"](modules/_runtx_.md)
-* ["state/index"](modules/_state_index_.md)
-* ["state/interface"](modules/_state_interface_.md)
-* ["state/stateManager"](modules/_state_statemanager_.md)
+* ["lib/evm/evm"](modules/_lib_evm_evm_.md)
+* ["lib/evm/opcodes/EIP1283"](modules/_lib_evm_opcodes_eip1283_.md)
+* ["lib/evm/opcodes/EIP2200"](modules/_lib_evm_opcodes_eip2200_.md)
+* ["lib/evm/opcodes/EIP2929"](modules/_lib_evm_opcodes_eip2929_.md)
+* ["lib/evm/opcodes/codes"](modules/_lib_evm_opcodes_codes_.md)
+* ["lib/evm/opcodes/functions"](modules/_lib_evm_opcodes_functions_.md)
+* ["lib/evm/opcodes/index"](modules/_lib_evm_opcodes_index_.md)
+* ["lib/evm/opcodes/util"](modules/_lib_evm_opcodes_util_.md)
+* ["lib/evm/precompiles/util/bls12_381"](modules/_lib_evm_precompiles_util_bls12_381_.md)
+* ["lib/exceptions"](modules/_lib_exceptions_.md)
+* ["lib/index"](modules/_lib_index_.md)
+* ["lib/runBlock"](modules/_lib_runblock_.md)
+* ["lib/runBlockchain"](modules/_lib_runblockchain_.md)
+* ["lib/runCall"](modules/_lib_runcall_.md)
+* ["lib/runCode"](modules/_lib_runcode_.md)
+* ["lib/runTx"](modules/_lib_runtx_.md)
+* ["lib/state/index"](modules/_lib_state_index_.md)
+* ["lib/state/interface"](modules/_lib_state_interface_.md)
+* ["lib/state/stateManager"](modules/_lib_state_statemanager_.md)
+* ["tests/BlockchainTestsRunner"](modules/_tests_blockchaintestsrunner_.md)
+* ["tests/GeneralStateTestsRunner"](modules/_tests_generalstatetestsrunner_.md)
+* ["tests/api/EIPs/eip-2537-BLS.spec"](modules/_tests_api_eips_eip_2537_bls_spec_.md)
+* ["tests/api/EIPs/eip-2929.spec"](modules/_tests_api_eips_eip_2929_spec_.md)
+* ["tests/api/berlin/eip-2315.spec"](modules/_tests_api_berlin_eip_2315_spec_.md)
+* ["tests/api/bloom.spec"](modules/_tests_api_bloom_spec_.md)
```

Comment:
hmmm looks like docs were somehow generated for the test files, although i am seeing it properly in the vm `typedoc.js` excludes, not sure where it might have gone wrong

Reply:
Ok, will continue with the release process, we can address this separately.